galaxy_ng/app/api/v3/serializers/sync.py

```diff
@@ -225,3 +232,36 @@ def get_last_sync_task_queryset(self, obj):
         return CollectionSyncTask.objects.filter(
             repository=obj.repository_set.order_by('-pulp_last_updated').first()
         ).first()
+
+
+def get_write_only_fields(serializer, obj, extra_data={}):
+    """
+    Returns a list of write only fields and whether or not their values are set
+    so that clients can tell if they are overwriting an existing value.
+    serializer: Serializer instance
+    obj: model object being serialized
+    extra_data: extra fields that might not be on obj. This is used when a write
+        only field is not one of the fields in the underlying data model.
+    """
+    fields = []
+
+    def _is_set(field_name):
+        if (field_name in extra_data):
+            return extra_data[field_name] is not None
```

Comment:
If you use `bool` you address the `""` and `None`
```suggestion
            return bool(extra_data[field_name])
```